cpp/include/cuspatial/haversine.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <rmm/mr/device/device_memory_resource.hpp>
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_view.hpp>
+#include <cuspatial/constants.hpp>
+ 
+namespace cuspatial {
+
+/**
+ * brief Compute Haversine distances between points in set A to the corrosponding points in set B.
```

Comment:
Note Haversine, unlike Hausdorff, is not a proper name, so shouldn't be capitalized (I just learned this!).

```suggestion
 * brief Compute haversine distances between points in set A to the corrosponding points in set B.
```
Also, can you add a link to the Wikipedia page or other reference so users know what this is calculating? (https://en.wikipedia.org/wiki/Haversine_formula)